README.md

```diff
@@ -16,63 +39,191 @@
 - [📖 About the Project](#about-project)
   - [🛠 Built With](#built-with)
     - [Tech Stack](#tech-stack)
-    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
 - [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
 <!-- PROJECT DESCRIPTION -->
 
 # 📖 [Hello-Microverse] <a name="about-project"></a>
 
+> Description
 
+**[Hello-Microverse]** is my first project in microverse. it is a way to help me learn how to make and submit a project at microverse
 
-**[Hello-microverse]** is my first project in microverse. it is a way to help me learn how to make and submit a project at microverse
-
-## 🛠 Built With <a name="built-with"></a>
+## 🛠 Built With <a name="built-with">HTML and CSS</a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
-
+> Describe the tech stack and include only the relevant sections that apply to your project.
 
 <details>
   <summary>Client</summary>
   <ul>
-    <li>Vanilla HTML and CSS</li>
+    <li><a href="https://en.wikipedia.org/wiki/HTML">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="">N/A</a></li>
   </ul>
 </details>
 
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="">N/A</a></li>
+  </ul>
+</details>
+<!-- GETTING STARTED -->
 
+## 💻 Getting Started <a name="getting-started"></a>
 
-<!-- Features -->
+> Describe how a new developer could make use of your project.
```

Comment:
[OPTIONAL] Please remove these instructions which are targeting you and add your own custom information.